packages/react/src/calendar/Calendar.tsx

```diff
@@ -16,6 +16,14 @@ import { CalendarWeekday } from "./CalendarWeekday";
 type CalendarProps = BoxProps<
   "div",
   {
+    /**
+     * When a selection mode is set, DayPicker will focus the first selected day
+     * (if set) or today's date (if not disabled).
+     *
+     * Use this prop when you need to focus DayPicker after a user action, for
+     * improved accessibility.
+     */
+    autoFocus?: boolean;
```

Comment:
I don’t think we need this to be a prop

just default it to true inside the calendar body